vsphere/internal/helper/virtualmachine/virtual_machine_helper.go

```diff
@@ -21,6 +21,25 @@ import (
 
 var errGuestShutdownTimeout = errors.New("the VM did not power off within the specified amount of time")
 
+// UUIDNotFoundError is an error type that is returned when a
+// virtual machine could not be found by UUID.
+type UUIDNotFoundError struct {
+	s string
```

Comment:
worth making this `message` to be self-explanatory?

Reply:
This is an internal field on a very small object - and this generally matches the pattern you see in something like the [`errors` package](https://golang.org/src/errors/errors.go), so I think it's fine :)